### ISSUE TITLE
JDK-8207957: TableSkinUtils should not contain actual code implementation

### ISSUE THREAD

**Issue by Maxoudela** [javafxports/openjdk-jfx#252]
Related JBS issues:
https://bugs.openjdk.java.net/browse/JDK-8207957

**Comment by abhinayagarwal** [comment]
I agree with [@kleopatra's comment](https://bugs.openjdk.java.net/browse/JDK-8207957?focusedCommentId=14217057&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14217057) that these methods shouldn't exists in TableView and should belong to either `TableColumn` or `TableColumnHeader`.

These changes also seem to be important to TableView2 as per [this comment](https://github.com/javafxports/openjdk-jfx/pull/163#issuecomment-438297212).
